### DISCUSSION TITLE
Teradata: stored procedure not visible in database navigator

### DISCUSSION THREAD

**Discussion by ahhmedbilal** [dbeaver/dbeaver#13584] [Q&A]
#### System information: 
- Operating system (distribution) and version: win 10
- DBeaver version: 21.1.2
- Java version 
- Additional extensions

#### Connection specification:
- Database name and version: Teradata 16.20.53.31
- Driver name: Teradata
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
in database navigator and under database name, i can see only "Tables" & " Views" node, but not any other node. Stored procedure and macro nodes are missing, can you plz add this feature. 
In Teradata Studio following nodes are available under database
1. Iindexes
2. Macros
3. Stored Procedure
4. Tables
5. User-Define Functions
6. Views. 
DBeaver should also show the same nodes. I am using the same jdbc driver in dbeaver as in TD studio. 

#### Steps to reproduce, if exist:
In database navigator, open any database, you will see "Tables" & " Views" node, but "Stored Procedure"

**Comment by LonwoLonwo** [comment]
Hello, @ahhmedbilal 
Can you please check your connection view settings? Is it advanced/custom?

![2021-08-19 09_41_33-DBeaver 21 1 0 - _PostgreSQL - dvdrental_ Script-33](https://user-images.githubusercontent.com/45152336/130020645-c2703105-532b-4637-b7ca-0ca99b4175f3.png)

And if it is advanced/custom, could you take a screen from Database Navigator Tree, please? 
The name/version of the Teradata driver will be also very helpful, thanks.

**Comment by ahhmedbilal** [comment]
Hi,

It was custom. I set it to Advance, now SP are visible.

Thanks
